pkg/kubectl/resource/builder_test.go

```diff
@@ -843,7 +846,7 @@ func TestIgnoreStreamErrors(t *testing.T) {
 	singular := false
 
 	err := b.Do().IntoSingular(&singular).Visit(test.Handle)
-	if err != nil || singular || len(test.Infos) != 2 {
+	if err == nil || singular || len(test.Infos) != 2 {
```

Comment:
@feihujiang, I just realized the PR will build if I check the code out with the old directory structure (github/GoogleCloudPlatforms/kubernertes...). I'll test it tomorrow. Sorry for the noise.